runtime/src/status_cache.rs

```diff
@@ -9,403 +9,229 @@ use std::ops::Deref;
 use std::ops::DerefMut;
 use std::sync::Arc;
 
-/// Each cache entry is designed to span ~1 second of signatures
-const MAX_CACHE_ENTRIES: usize = solana_sdk::timing::MAX_HASH_AGE_IN_SECONDS;
-
-type FailureMap<T> = HashMap<Signature, T>;
-
-#[derive(Clone)]
-struct Status<T> {
+// Blockhash found in the transaction
+type TransactionBlockHash = Hash
```

Comment:
`Blockhash` is one word. And no need to qualify a type with the name of one container it might be found in.

Reply:
type alias seems better than a comment